ribasim_qgis/core/nodes.py

```diff
@@ -151,7 +137,9 @@ def layer_from_geopackage(self) -> QgsVectorLayer:
 
     def from_geopackage(self) -> tuple[QgsVectorLayer, Any, Any]:
         self.layer_from_geopackage()
-        return (self.layer, self.renderer, self.labels)
+        fn = STYLE_DIR / f"{self.input_type().replace(' / ', '_')}Style.qml"
```

Comment:
I see `STYLE_DIR = Path(__file__).parent / "styles"`. I don't see the styles there, only in Ribasim Python. How does that work?

Reply:
In the pixi task where we install the qgis plugin, it symlinks the styles folder.

Reply:
Somebody else might just install the plugin as a zip, then the symlink would be missing? Or do I miss something?

Reply:
I've now added the pixi (symlink) command to our Teamcity CI, so release zips contain the styles.